system-parachains/encointer/Cargo.toml

```diff
@@ -83,21 +79,21 @@ sp-genesis-builder = { workspace = true }
 pallet-xcm = { workspace = true }
 polkadot-parachain-primitives = { workspace = true }
 polkadot-runtime-common = { workspace = true }
-xcm = { package = "staging-xcm", default-features = false, version = "8.0.1" }
-xcm-builder = { package = "staging-xcm-builder", default-features = false, version = "8.0.1" }
-xcm-executor = { package = "staging-xcm-executor", default-features = false, version = "8.0.1" }
+xcm = { workspace = true }
+xcm-builder = { workspace = true }
+xcm-executor = { workspace = true }
 
 # Cumulus dependencies
 cumulus-pallet-aura-ext = { workspace = true }
 cumulus-pallet-parachain-system = { features = [
-  "parameterized-consensus-hook",
+	"parameterized-consensus-hook",
 ], workspace = true }
 cumulus-pallet-xcm = { workspace = true }
 cumulus-pallet-xcmp-queue = { workspace = true }
 cumulus-primitives-aura = { workspace = true }
 cumulus-primitives-core = { workspace = true }
 cumulus-primitives-utility = { workspace = true }
-parachain-info = { package = "staging-parachain-info", default-features = false, version = "0.8.0" }
```

Comment:
I think the workspace dependencies feature is cool. The potential problem I see is with Encointer and its pallets, which use a different version of the Polkadot SDK. When we upgrade to a new version of the Polkadot SDK, Encointer is unlikely to compile. As a result, we won't be able to create a new release unless Encointer updates its pallets to the same Polkadot SDK version.

Therefore, it's questionable whether using workspace dependencies is a good idea for Encointer at this time. But let's give it a try and see what happens.

Reply:
Okay i did not think about this. It basically means that we need to update the encointer pallets in advance to the version of the SDK that we plan to use next.